javafx-archetype-fxml/src/main/resources/archetype-resources/src/main/java/module-info.java

```diff
@@ -1,7 +1,13 @@
+#if ("8" == ${javafx-version} || "1.8" == ${javafx-version})
+/* doesn't work with source level ${javafx-version}:
```

Comment:
Does `${javafx-version}` gets replaced by its value in the comment?

Reply:
Yes. From a test I got:
```
/* doesn't work with source level 8:
module com.raelity.play.mp.sim. {
    requires javafx.controls;
    exports com.raelity.play.mp.sim.;
}
*/
```